CoreWiki.Test/Pages/AllPageTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿using CoreWiki.Models;
+using CoreWiki.Pages;
+using Microsoft.EntityFrameworkCore;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace CoreWiki.Test.Pages
+{
+	public class AllPageTests
+	{
+
+		[Fact]
+		public async Task OnGet_PopulatesThePageModel_WithAListOfArticles()
+		{
+
+			// Arrange
+			var optionsBuilder = new DbContextOptionsBuilder<InMemoryDbContext>();
```

Comment:
Instead of creating a new instance of the db for each test, should this be a shared context amongst all the tests?

Reply:
Yes.  I have already done just that, but first we need to completely refactor database contexts from all pages/components/helpers.

Reply:
Feel free to PR my "tests" fork.